packages/tools/guiEditor/src/components/propertyTab/propertyGrids/gui/commonControlPropertyGridComponent.tsx

```diff
@@ -161,6 +198,54 @@ export class CommonControlPropertyGridComponent extends React.Component<ICommonC
         }
     }
 
+    public addVal = (newVal: { label: string; value: number }, prevVal: number) => {
+        if (newVal.label === "") {
+            return;
+        }
+        (async () => {
+            await document.fonts.ready;
+            let displayVal = false;
+            const fonts = JSON.parse(String(window.sessionStorage.getItem("fonts")));
+            if (!(fonts.find((element: IInspectableOptions) => element.label.toLowerCase() === newVal.label.toLowerCase()) === undefined)) {
+                alert("This font is already available");

```

Comment:
@PatrickRyanMS - are we fine with alerts? or do we have a different system for messages (like a toast at the bottom?)

Reply:
I asked Patrick about it and I believe he said that alerts were the best option for this!